examples/snake/ws-resources.json

```diff
@@ -3,6 +3,10 @@
     "/index.html": {
       "Content-Type": "text/html; charset=utf-8",
       "Cache-Control": "max-age=3500"
+    },
+    "/*.svg": {
```

Comment:
Question: this is actually more generica than what we have for the routes. Do we want more expressive regexes such as this one? The routes only allow a wildcard at the end

Reply:
In the case of headers there will be cases that the headers will be defined based on the file extension. 

Regarding routing, we use the same trick of replacing the wildcard `*` with the regex `.*` so this way of more expressive paths _is_ already supported there too.

Reply:
>Regarding routing, we use the same trick of replacing the wildcard * with the regex .* so this way of more expressive paths is already supported there too.

of course

> So I would say this expressiveness would be needed.

makes sense.  However, we then have a problem with the "longest" matching. We cannot use the length of the string anymore.

for example, let's say we have:

- `/*.gnuplot`
- `/a/b/c/*`

both match `/a/b/c/plot.gnuplot`, and the former is a _shorter_ path, but actually it's a long string, so it matches first, leading to a bug in the behavior

Reply:
I updated the code so that there are unit tests to catch cases like the one that you discovered. 

To fix the longest path matching, I thought of including the matching pattern with the most slashes `'/'`:

```rust
matched_pattern.max_by_key(|(path, _)| path.split('/').count()) // Longest path match
```

The issue should be resolved now.

Reply:
It just came to mind that I didn't include the case of windows platforms that uses front slash instead of backslash. Need to account for this too!

Reply:
> It just came to mind that I didn't include the case of windows platforms that uses front slash instead of backslash. Need to account for this too!

uhm have you ever tried the site builder on windows? even without this modification, I'm not sure it would work.

Reply:
>You mean that the site-builder on windows doesn't work in general?

yes, i don't think it does